alertlogic/service.py

```diff
@@ -18,6 +18,18 @@ def set_session(self, session):
         """
         self._session = session
 
+    def get(self, path_parts, params=None, json=None):
```

Comment:
cosmetic: there is no limitation here on what payload in "json" could be. I would either call it "payload", or added json.dumps(json) if we always want to make sure json is passed to our services.

Reply:
ahh i take this one back. requests are doing this.